### ISSUE TITLE
The ability to change the colour of the accessibility focus.

### ISSUE THREAD

**Issue by mdr** [flutter/flutter#44359]
## Use case

Anywhere where the background is close in colour to the accessibility focus, or else has insufficient contrast, makes it hard to see which element is focussed.

## Proposal

A widget can specify an accessibility focus colour. In addition, it would be useful to be able to specify an accessibility focus colour in the theme.

**Comment by jonahwilliams** [comment]
By the accessibility focus I assume you mean the Talkback or VoiceOver outline? That is not drawn by Flutter and isn't customizable.

**Comment by mdr** [comment]
> By the accessibility focus I assume you mean the Talkback or VoiceOver outline?

Yeah, exactly.

> That is not drawn by Flutter and isn't customizable.

Ah, I see. Is that just a limitation in the mobile platforms themselves?

**Comment by jonahwilliams** [comment]
For the most part  - they run as separate programs. on Android you could probably customize it by building your own TalkBack, though a application would still have no way to interact with it

**Comment by goderbauer** [comment]
Closing this since flutter doesn't have control over this as per previous comment.
